.env.production

```
@@ -1,5 +1,5 @@
 # Production Environment
-VITE_API_BASE_URL=https://backend.nasam.co/
+VITE_API_BASE_URL=https://backend.nasam.co/v1
 VITE_API_KEY=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
 VITE_PAYMENTS_KEY=pk_live_MYiRnhso1gAongtiZeNtGQwbVS6qpRkkbJXwFwRZ
 VITE_ENVIRONMENT=production
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_ENVIRONMENT key should go before the VITE_PAYMENTS_KEY key

(UnorderedKey)